src/components/molecules/VenueRoomItem/VenueRoomItem.scss

```diff
@@ -0,0 +1,22 @@
+@import "scss/constants.scss";
+
+.VenueRoomItem {
+  padding: $spacing--md;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+  &:hover {
+    background-color: $saturated-black;
+  }
+
+  &__room-icon {
+    height: 34px;
+    width: 34px;
```

Comment:
room-icon-size

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1583/commits/f8fa0c67c34ac626919737b4bbd112b4bd715553